internal/collectors/architecture.go

```diff
@@ -0,0 +1,19 @@
+package collectors
+
+import "github.com/SUSE/connect-ng/internal/util"
+
+type Architecture struct{}
+
+func (Architecture) run(arch string) (Result, error) {
+	// For now just a playback of the already existing value but in future
```

Comment:
This confused me at first. We should write that the given `arch` actually comes from calling `DetectArchitecture` beforehand. Hence, I would also write the origin of this value, and then adding the note as it is that in the future we might further expand this info.

Reply:
do you think it makes sense to actually call `DetectArchitecture` again?

Reply:
Not really, because we really need to know the arch before collecting anything, right? So it's fine, but we should document it anyways just so a newcomer is not confused in this file.

Reply:
I updated the comments above mentioning that `DetectArchitecture` is called before already